src/multiqueue.py

```diff
@@ -1,6 +1,5 @@
 """
-src/multiqueue.py
-=================
+Manage multiple queues
```

Comment:
A queue with multiple internal subqueues. Elements are added into a random subqueue, and retrieval rotates

Reply:
Okay. Thank you